From 5da88d4a9ffb5248ac036df5361f4eb280c585a8 Mon Sep 17 00:00:00 2001
From: Yehuda Yitschak <yehuday@marvell.com>
Date: Thu, 7 Jan 2016 10:37:06 +0200
Subject: [PATCH 0650/1240] doc: updated the board options table for A8K

Added CP prefix to SPI and UART to differentiate between AP and CP

Change-Id: Ib85c191804a9141e2b4bf58e84978d1fc175b6aa
Signed-off-by: Yehuda Yitschak <yehuday@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/26615
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 doc/mvebu/a8k/board_options.txt | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/doc/mvebu/a8k/board_options.txt b/doc/mvebu/a8k/board_options.txt
index a829577..700d787 100644
--- a/doc/mvebu/a8k/board_options.txt
+++ b/doc/mvebu/a8k/board_options.txt
@@ -17,13 +17,13 @@ SerDes:	lane0	| sgmii2	| pex_x4_l0	| pex_x4_l0	| pex_x4_l0	| pex_x4_l0	| sata1
 	lane4	| usb3h1	| pex_x1_0	| pex_x1_0	| usb3h1	| usb3h1	| rxauil0	| rxauil0	|
 	lane5	| pex_x1_1	| pex_x1_1	| pex_x1_1	| pex_x1_1	| pex_x1_1	| rxauil1	| rxauil1	|
 ---------------------------------------------------------------------------------------------------------------------------------
-MPPs: 	SPI0	| N/C		| N/C		| N/C		| N/C		| N/C		| N/C		| N/C		|
-	SPI1	| [13-16]	| [13-16]	| [13-16]	| N/C		| [13-16]	| [13-16]	| [13-16]	|
+MPPs: 	CP-SPI0	| N/C		| N/C		| N/C		| N/C		| N/C		| N/C		| N/C		|
+	CP-SPI1	| [13-16]	| [13-16]	| [13-16]	| N/C		| [13-16]	| [13-16]	| [13-16]	|
 	NAND	| N/C		| N/C		| N/C		| [13,17-27]	| N/C		| N/C		| N/C		|
 	RGMII0	| N/C		| [0-12]	| [0-12]	| [0-12]	| [0-12]	| N/C		| N/C		|
 	RGMII1	| N/C		| [44-55]	| [44-55]	| [44-55]	| [44-55]	| N/C		| N/C		|
-	UART0	| [29-30]	| [29-30]	| [29-30]	| [29-30]	| [29-30]	| [29-30]	| [29-30]	|
-	UART1	| [40-41]	| [40-41]	| [40-41]	| [40-41]	| [40-41]	| [40-41]	| [40-41]	|
+	CP-UART0| [29-30]	| [29-30]	| [29-30]	| [29-30]	| [29-30]	| [29-30]	| [29-30]	|
+	CP-UART1| [40-41]	| [40-41]	| [40-41]	| [40-41]	| [40-41]	| [40-41]	| [40-41]	|
 	SD	| [56-62]	| [56-62]	| [56-62]	| [56-62]	| [56-62]	| [56-62]	| [56-62]	|
 	TDM	| [0-11]	| N/C		| N/C		| N/C		| N/C		| N/C		| N/C		|
 	TWSI	| [37-38]	| [37-38]	| [37-38]	| [37-38]	| [37-38]	| [37-38]	| [37-38]	|
-- 
1.9.1

